spec/matchers/have_run.rb

```diff
@@ -0,0 +1,25 @@
+RSpec::Matchers.define :have_run do |job_name|
+  match do |clock_test|
+    @times_run ||= 1
+    clock_test.manager.times_run(job_name) == @times_run
```

Comment:
I think it'd be more beneficial if, by default, it didn't check if it ran only once, but more than 0 times.

If exactly is chained to it, __then__ check if `clock_test.manager.times_run(job_name) == @times_run`.